public/css/wyr-single.css

```diff
@@ -42,4 +42,22 @@
   100% {
     transform: translateX(-150%);
   }
-}
+} */
+
+    #add-question-modal {
+      background-color: rgba(0, 0, 0, 0.5);
+    }
+
+    #add-question-modal.show {
+      animation: fadeIn 0.3s;
+    }
+
+    @keyframes fadeIn {
+      from {
+        opacity: 0;
+      }
+
+      to {
+        opacity: 1;
+      }
+    }
```

Comment:
this is additional styling for giving the #add-question-modal a faded background color. There is a slight animation to bring it into view (same as the newComment modal)